requirements.txt

```diff
@@ -24,3 +24,4 @@ jinja2==3.0.2
 itsdangerous==2.0.1
 google-api-python-client==2.56.0
 google-cloud-storage==2.5.0
+cffi==1.15.1
```

Comment:
What is this requirement for?

Reply:
python-cffi packages are required by the platform, which are not installed along with python, we have to install them explicitly, I received the following error [No module named _cffi_backend](https://stackoverflow.com/questions/34370962/no-module-named-cffi-backend).